internal/orchestrator/encryption_exported_test.go

```
@@ -175,14 +175,13 @@ func TestPrepareAgeRecipients_NoRecipientsNonInteractiveErrors(t *testing.T) {
 	if err != nil {
 		t.Fatalf("pipe stdin: %v", err)
 	}
+	defer inR.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `inR.Close` is not checked

(errcheck)